app/styles/_variables.scss

```diff
@@ -410,6 +410,10 @@ $overlay-background-color: rgba(0, 0, 0, 0.4);
 
   /** rebase progress bar */
   --dialog-rebase-progress-background: #{$green};
+
+  --status-pending-color: #{$yellow-700};
+  --status-error-color: #{$red-500};
+  --status-success-color: #{$green-500};
```

Comment:
Could you please copy and paste these over to `_dark.scss` as well so that they can be modified independently? The colors are fine for the dark theme for now but I'd like to maintain it as an independent stylesheet.

Reply:
Added in 21f7a8702090a35e907f12cb4cf2df63a24a2ef6